x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/fake_data.ts

```diff
@@ -19,6 +21,12 @@ const dateOffsets = [
 
 const randomNumbers = [5, 50, 500, 5000, 50000];
 
+interface SavedFakePolicyData {
+  items: PolicyData[];
+}
+
+let savedFakePolicyData: SavedFakePolicyData = { items: [] };
```

Comment:
hacky stuff for the fake data, just wanna maintain the random set of data so I can use ids from the URL to find a Policy record